src/components/templates/ReactionPage/ReactionList.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo, useState } from "react";
+import classNames from "classnames";
+
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+
+import {
+  chatMessageAsMessageToTheBand,
+  Reaction,
+  ReactionsTextMap,
+} from "utils/reactions";
+import { withId, WithId } from "utils/id";
+
+import { User } from "types/User";
+import { ChatMessage } from "types/chat";
+
+import { useWorldUsersByIdWorkaround } from "hooks/users";
+
+import UserProfileModal from "components/organisms/UserProfileModal";
+import UserProfilePicture from "components/molecules/UserProfilePicture";
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+export interface ReactionListProps {
+  reactions: Reaction[];
+  chats: ChatMessage[];
+  small?: boolean;
+}
+
+export const ReactionList: React.FC<ReactionListProps> = ({
+  reactions,
+  chats,
+  small = false,
+}) => {
+  // @debt see comments in useWorldUsersByIdWorkaround
+  const { worldUsersById } = useWorldUsersByIdWorkaround();
+
+  const [selectedUserProfile, setSelectedUserProfile] = useState<
+    WithId<User>
+  >();
+
+  const allReactions = useMemo(() => {
+    const chatsAsBandMessages = chats?.map(chatMessageAsMessageToTheBand) ?? [];
```

Comment:
wrap into useMemo?

Reply:
Wrap what into useMemo? There’s already a useMemo on this line?

Reply:
missed that^